registry/registryprovider.go

```diff
@@ -349,7 +349,7 @@ func ShortTypeToDownloadURLs(rp RegistryProvider, t string) ([]string, Registry,
 	}
 
 	if r == nil {
-		panic(fmt.Errorf("cannot get github registry for %s", t))
+		return nil, nil, fmt.Errorf("cannot get github registry for %s", t)
```

Comment:
Add the TODOs. The registry.go panic must be intentionally chosen by the caller, and the panic in deployments.go is not cross package, so both seem reasonable to me. PTAL.